client/my-sites/checkout/upsell-nudge/style.scss

```diff
@@ -24,6 +27,54 @@
 	}
 }
 
+.upsell-nudge__price-placeholder {
+	width: 33%;
+	margin: 0 auto;
+	height: 20px;
+}
+
+.upsell-nudge__hold-tight-placeholder {
+	width: 33%;
+	margin-left: auto;
+	margin-right: auto;
+	margin-bottom: 20px;
+	height: 20px;
+}
+
+.upsell-nudge__form-placeholder {
+	width: 100%;
+	margin-top: 64px;
+	float: left;
```

Comment:
Can we use flex instead to achieve the same?

Reply:
It was not needed at all this styling, I've removed it.